lib/manager/terraform/lockfile/index.spec.ts

```diff
@@ -0,0 +1,411 @@
+
+    mockGetPkgReleases
+      .mockResolvedValueOnce(
+        new Promise((resolve) =>
+          resolve({
+            // aws
+            releases: [
+              {
+                version: '2.30.0',
+              },
+              {
+                version: '3.0.0',
+              },
+            ],
+          })
+        )
+      )
+      .mockResolvedValueOnce(
+        new Promise((resolve) =>
+          resolve({
+            // azurerm
+            releases: [
+              {
+                version: '2.50.0',
+              },
+            ],
+          })
+        )
+      )
+      .mockResolvedValueOnce(
+        new Promise((resolve) =>
+          resolve({
+            // random
+            releases: [
+              {
+                version: '2.2.1',
+              },
+            ],
+          })
+        )
+      );
+    mockHash.mockResolvedValue([
+      'h1:lDsKRxDRXPEzA4AxkK4t+lJd3IQIP2UoaplJGjQSp2s=',
+      'h1:6zB2hX7YIOW26OrKsLJn0uLMnjqbPNxcz9RhlWEuuSY=',
+    ]);
+
+    const localConfig: UpdateArtifactsConfig = {
+      updateType: 'lockFileMaintenance',
+      ...config,
+    };
+    const result = await updateArtifacts({
+      packageFileName: '',
+      updatedDeps: [],
+      newPackageFileContent: '',
+      config: localConfig,
+    });
+    expect(result).toBeNull();
+
+    expect(mockHash.mock.calls).toBeArrayOfSize(0);
+    expect(mockHash.mock.calls).toMatchSnapshot();
+  });
+
+  it('return null if hashing fails', async () => {
+    fs.readLocalFile.mockResolvedValueOnce(validLockfile as any);
+
+    mockGetPkgReleases
+      .mockResolvedValueOnce(
+        new Promise((resolve) =>
+          resolve({
+            // aws
+            releases: [
+              {
+                version: '2.30.0',
+              },
+              {
+                version: '3.0.0',
+              },
+              {
+                version: '3.36.0',
+              },
+            ],
+          })
+        )
+      )
+      .mockResolvedValueOnce(
+        new Promise((resolve) =>
+          resolve({
+            // azurerm
+            releases: [
+              {
+                version: '2.50.0',
+              },
+              {
+                version: '2.55.0',
+              },
+              {
+                version: '2.56.0',
+              },
+            ],
+          })
+        )
+      )
+      .mockResolvedValueOnce(
+        new Promise((resolve) =>
+          resolve({
+            // random
+            releases: [
+              {
+                version: '2.2.1',
+              },
+              {
+                version: '2.2.2',
+              },
+              {
+                version: '3.0.0',
+              },
+            ],
+          })
+        )
+      );
+    mockHash.mockResolvedValue(null);
+
+    const localConfig: UpdateArtifactsConfig = {
+      updateType: 'lockFileMaintenance',
+      ...config,
+    };
+
+    process.env.RENOVATE_TERRAFORM_LOCK_FILE = 'test';
```

Comment:
```suggestion
    process.env.RENOVATE_X_TERRAFORM_LOCK_FILE = 'test';
```
Wouldn't be easier to set in beforeEach once for all tests?